api/v1alpha1/common.go

```diff
@@ -179,40 +180,39 @@ func UpdateCondWithFixedLen(l int, conds *ConditionedStatus, cond Condition) {
 	conds.Conditions = append(conds.Conditions, cond)
 }
 
+type FilterFunc func(FilterCond, string) bool
+
 type Filter struct {
```

Comment:
`Filter` is not a good name since this seems to define the content which will be filtered.